apps/chat/src/components/Chat/ChangePathDialog.tsx

```diff
@@ -57,7 +57,7 @@ export const ChangePathDialog = ({
   const [isAllFoldersOpened, setIsAllFoldersOpened] = useState(true);
   const [openedFoldersIds, setOpenedFoldersIds] = useState<string[]>([]);
   const [selectedFolderId, setSelectedFolderId] = useState<string | undefined>(
-    '',
+    getRootId(),
```

Comment:
is `ChangePathDialog` used only for `files`?

Reply:
Yes